ngrinder-frontend/src/js/components/agent/List.vue

```diff
@@ -36,123 +36,137 @@
+            :data-manager="dataManager"
+            @vuetable:pagination-data="beforePagination">
+
+            <template slot="state" slot-scope="props">
+                <div class="ball" data-html="true" rel="popover">
+                    <img class="status" :src="`/img/ball/${props.rowData.state.iconName}`"/>
+                </div>
+            </template>
+
+            <template slot="domain" slot-scope="props">
+                <router-link :to="{ name: 'agentDetail', params: { agentId: props.rowData.id, agentProp: props.rowData } }"
+                             :value="props.rowData.ip" v-text="props.rowData.ip">
+                </router-link>
+            </template>
+
+            <template slot="approved" slot-scope="props">
+                <div class="btn-group">
+                    <button class="btn btn-primary disapproved"
+                            :class="{ active: !props.rowData.approved }"
+                            v-text="i18n('agent.list.disapproved')" @click="disapprove(props.rowData)">
+                    </button>
+                    <button class="btn btn-primary approved"
+                            :class="{ active: props.rowData.approved }"
+                            v-text="i18n('agent.list.approved')" @click="approve(props.rowData)">
+                    </button>
+                </div>
             </template>
-            <tr v-show="agents.length === 0">
-                <td colspan="8" class="center" v-text="i18n('common.message.noData')"></td>
-            </tr>
-            </tbody>
-        </table>
-        <!-- TODO: Paginate using datatables -->
+
+        </vuetable>
+        <vuetable-pagination
+            ref="pagination"
+            :css="table.css.pagination"
+            @vuetable-pagination:change-page="changePage">
+        </vuetable-pagination>
     </div>
 </template>
 
 <script>
     import Component from 'vue-class-component';
     import { Mixins } from 'vue-mixin-decorator';
+    import _ from 'lodash';
+    import Vuetable from 'vuetable-2';
+    import VuetablePagination from 'vuetable-2/src/components/VuetablePagination.vue';
     import Base from '../Base.vue';
+    import TableConfig from './mixin/TableConfig.vue';
     import MessagesMixin from '../common/mixin/MessagesMixin.vue';
 
     @Component({
         name: 'agentList',
+        components: { Vuetable, VuetablePagination },
     })
-    export default class AgentList extends Mixins(Base, MessagesMixin) {
+    export default class AgentList extends Mixins(Base, MessagesMixin, TableConfig) {
+        agents = [];
         regions = [];
         region = '';
 
-        agents = [];
-
         downloadLink = '';
+        updateStatesTimer = null;
 
-        page = {
-            number: 1,
-            totalPages: 1,
-            size: 10,
+        table = {
+            css: {},
+            renderingData: {
+                data: [],
+                pagination: {
+                    per_page: 10,
```

Comment:
why does it have different snake case?

Reply:
vuetable-2 uses snake case like this
https://www.vuetable.com/guide/pagination.html#how-the-pagination-component-works

so I use snake case to table related value.
change to using the camel case?

Reply:
@junoyoon 
Changed from snake case to carmel case for all tables.
https://github.com/naver/ngrinder/pull/452/commits/53ce08a4746be2545ec376c0cf3457f5dcd35fee#diff-31ce3a93059eae7178230badd1502f68R111